py/visdom/extra_deps/__init__.py

```diff
@@ -1,5 +0,0 @@
-# Copyright 2017-present, The Visdom Authors
```

Comment:
Why do we drop this file? The core function of it is to retain the directory, wherein people can import optional dependencies for Visdom. Realistically, this is only useful for the `bhtsne` library, but that may still be in use. Perhaps instead of just an `__init__.py`, it makes sense to have a `README.md` too, but that's out of scope for this PR.

Reply:
Ah, didn't know that. I've removed the deletion in the latest push below.

Reply:
Hm I'm not seeing the most recent push. I'll merge this in once it's there!

Reply:
Yes, I totally forgot to push, but it should be there now.